### PR TITLE
Send secret rotation messages.

### PR DESCRIPTION
**Pull request by MancunianSam**
This will send messages to slack from the tdr-rotate-keycloak-secrets
lambda.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,23 @@ This project is for sending slack/email/SQS messages in response to events. It s
...
+* Keycloak secret rotation messages. When the Keycloak secret rotation lambda runs, it will send success or failure messages to the notifications SNS topic.
 
 ## Run locally
 
 Set these environment variables, either on the command line or in IntelliJ depending on how you want to run the app:
 
 * `SLACK_WEBHOOK`: the webhook URL of a Slack app. You can [create a new app][Slack-app] (steps 1-3) in Slack for testing purposes.
-  Use the `#bot-testing` channel rather than a team channel to avoid confusion and spam.
+  Use the `#bot-testing` channel rather than a team channel to avoid confusion and spam. This webhook goes to the #da-tdr-notifications channel.
+* `SLACK_JUDGMENT_WEBHOOK` This webhook publishes to the #da-tdr-prod-exports-judgments channel.
```
Comment (TomJKing) [inline README.md:19]: Could you just add an additional comment, that it should only be used to send production judgment transfer notifications?

**Review by TomJKing — COMMENTED** [review]
Looks good. Thanks. Just one very minor comment

File: README.md (hunk 1 above)
Comment (MancunianSam) [reply README.md]: Ah good point. I've added it in.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**